protocol/avdl/keybase1/ephemeral.avdl

```diff
@@ -7,38 +7,42 @@ protocol ephemeral {
   @typedef("int64")
   record EkGeneration {}
 
+  ////////////////////////////////////////////////////////////////////////
+
+  record DeviceEkMetadata {
+    @jsonkey("device_ephemeral_dh_public")
+    KID kid;
+    @jsonkey("hash_meta")
+    HashMeta hashMeta;
+    EkGeneration generation;
+    Time ctime;
+  }
+
   // NOTE: The device/user/team structs are very similar here, so it might seem
   // like a good idea to unify them. However! One thing we have to be careful
   // with is that signatures of one type don't get confused for another type.
   // Thus we use distinct field names like "device_ephemeral_dh_public" and
   // "user_ephemeral_dh_public". If anyone ever tries to refactor all these
   // structs, please be careful with this detail.
-  record DeviceEkMetadataStatement {
+  record DeviceEkStatement {
```

Comment:
Is this a revert? I'm fine with it either way, just making sure.

Reply:
yeah i think `DeviceEkStatement` is most concise. the inner values are still called `metadata` instead of just `ek` but `statement` is the abstraction these objects encapsulate